### PR TITLE
Temporarily disable unit tests causing build failures.

### PR DESCRIPTION
**Pull request by cesarbs**
@JunTaoLuo

### CODE REVIEW

File: test/Microsoft.Dnx.Compilation.CSharp.Tests/CompilationContextFacts.cs [hunk 1]
```diff
@@ -13,59 +13,60 @@ namespace Microsoft.Dnx.Compilation.CSharp.Tests
 {
     public class CompilationContextFacts
     {
-        [Fact]
-        public void DefaultConstructorInitializeCorrectly()
-        {
-            var compilation = CSharpCompilation.Create("fakecompilation");
-            var fakeProject = CreateMockProject();
+        // Temporarily disabling until bootstrapping issue is resolved.
```
Comment (moozzyk) [inline test/Microsoft.Dnx.Compilation.CSharp.Tests/CompilationContextFacts.cs]: Use skip

**Comment by cesarbs** [comment]
Opened #3014 to track this.

**Comment by cesarbs** [comment]
@victorhurdugaci

**Comment by victorhurdugaci** [comment]
:shipit: after Travis passes

**Pull request merged**